### PR TITLE
a few fixes by ruff

### PR DESCRIPTION
**Pull request by fchapoton**
as suggested and done by the linter `ruff`

### CODE REVIEW

**Comment by mmatera** [comment]
Thanks! LGTM

**Comment by mmatera** [comment]
OK, something went wrong. @fchapoton, please run black over the changed code, and lets try to detect why tests does not pass now.

File: mathics/builtin/patterns/composite.py [hunk 1]
```diff
@@ -75,8 +75,7 @@ def get_match_count(
             if range_lst is None:
                 range_lst = tuple(sub)
```
Comment (mmatera) [inline mathics/builtin/patterns/composite.py:76]: change this by 
`range_lst = list(sub)`

**Comment by fchapoton** [comment]
which version of black ?

**Comment by rocky** [comment]
Good - thanks!

**Pull request merged**